src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/HPackDecoder.cs

```diff
@@ -27,6 +27,13 @@ private enum State : byte
             DynamicTableSizeUpdate
         }
 
+        private struct HeaderData
```

Comment:
A better name is welcomed.

Reply:
Maybe `HeaderField`?

https://httpwg.org/specs/rfc7541.html#header.representation